### PR TITLE
IMP readme for ubuntu installation

### PR DESCRIPTION
**Pull request by quentinDupont**
maybe there's more things obsolete to delete, but that's a start

### CODE REVIEW

**Review by legalsylvain — APPROVED** [review]
thanks ! @PierrickBrun could you take a look ?

File: README.md [hunk 1]
```diff
@@ -190,12 +174,8 @@ sslkey=c:/pywebdriver/localhost+2-key.pem
 
 ## <a name="browser-settings"></a>Browser settings
 
-You need to confirmed security exception on your browers for the following paths:
-http://localhost and https://localhost
-
-see
-<a href="https://support.mozilla.org/en-US/kb/what-does-your-connection-is-not-secure-mean">here</a>
```
Comment (PierrickBrun) [inline README.md:197]: The link to the firefox documentation may be useful

File: README.md [hunk 2]
```diff
@@ -93,34 +93,18 @@ each file or in a readme file of the folder. The main other contributors are:
...
-
-If you update from the OLD version, you will need to uninstall nginx after upgrading or
+If you update from an OLD version, you will need to uninstall nginx after upgrading or
 you could face port reservation problems. Also it is not required anymore so there is no
 reason to have it.
 
-## <a name="install-script"></a>Installation on Ubuntu 18.04 with a signed certificate using mkcert
-
-Alternatively you can use the script
-[debian/install.sh](https://github.com/akretion/pywebdriver/tree/master/debian/install.sh).
```
Comment (PierrickBrun) [inline README.md:120]: The script is still relevant, maybe the text should be changed but we should not just remove it

**Review by PierrickBrun — COMMENTED** [review]
Thank you for this work !

File: README.md [hunk 3]
```diff
@@ -242,17 +222,6 @@ sslkey=privkey.pem
 
 ```
 
-#### <a name="browser-settings"></a>Epson OCD300
-
-For the customer display **Epson OCD300**, you need to change the config file
```
[the remark this answers is not in the document]
Comment (quentinDupont) [reply README.md:247]: I thought it was automatic in the last version ?

File: README.md (hunk 2 above)
Comment (quentinDupont) [reply README.md:120]: Yes i keep it actually, I put together the two parts of the readme (for ubuntu) into one part, so there is only one way to install pywebdriver, with this script :)

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (quentinDupont) [reply README.md:247]: Hello, I reinstalled pywebdriver, and it works like a charm with OCD300 without changing parameters in config, and letting this :
```
[display_driver]
; driver_name=epson
device_name=auto
```

**Review by PierrickBrun — APPROVED** [review]

**Pull request merged**